test/EndToEnd/vs.ps1

```diff
@@ -303,6 +303,15 @@ function New-ClassLibrary {
     New-Project ClassLibrary $ProjectName $SolutionFolderName
 }
 
+function New-ClassLibrary46 {
```

Comment:
please change the name to something meaningful. class library 46 makes no sense to me.